`import a, * as b from "a"` fails to print at all

Input:

```js
import a, * as b from "a"
```

Output:

```js
prettier.format 1 error:
AssertionError: false == true
    at genericPrintNoParens (/home/lydell/forks/prettier/src/printer.js:417:16)
    at genericPrint (/home/lydell/forks/prettier/src/printer.js:41:28)
    at comments.printComments.p (/home/lydell/forks/prettier/src/printer.js:2406:12)
    at Object.printComments (/home/lydell/forks/prettier/src/comments.js:293:17)
    at printGenerically (/home/lydell/forks/prettier/src/printer.js:2404:21)
    at /home/lydell/forks/prettier/src/printer.js:1558:25
    at FastPath.map (/home/lydell/forks/prettier/src/fast-path.js:153:19)
    at printStatementSequence (/home/lydell/forks/prettier/src/printer.js:1543:8)
    at /home/lydell/forks/prettier/src/printer.js:126:20
    at FastPath.call (/home/lydell/forks/prettier/src/fast-path.js:99:16)
```

https://jlongster.github.io/prettier/#%7B%22content%22%3A%22import%20a%2C%20*%20as%20b%20from%20%5C%22a%5C%22%5Cn%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3Afalse%2C%22bracketSpacing%22%3Atrue%2C%22doc%22%3Afalse%7D%7D

Found using the fuzzer in #371 (using the newly added shift-fuzzer)

--- a/src/printer.js
+++ b/src/printer.js
@@ -415,10 +415,8 @@ function genericPrintNoParens(path, options, print) {
           "specifiers"
         );
 
-        assert.ok(standalones.length <= 1);
-
         if (standalones.length > 0) {
-          parts.push(standalones[0]);
+          parts.push(join(", ", standalones));
         }
 
         if (standalones.length > 0 && grouped.length > 0) {

--- a/tests/import/multiple_standalones.js
+++ b/tests/import/multiple_standalones.js
@@ -1,0 +1,1 @@
+import a, * as b from 'a';

